content/components/selectpanel.mdx

```diff
@@ -164,9 +149,54 @@ It is important to provide visual cues to the user when certain processes may ta
 
 By employing these loading states, users can have a better understanding of the process and remain informed throughout their interactions with the component.
 
-## Best practices
+## Behavior
 
-### Be predictable
+### Triggers
+
+<img
+  width="960"
+  alt="Various trigger examples are displayed. The first example showcases an icon button with a cog icon. The second example presents a button with the internal label 'Assignees: 2 people'. The third example features a button with the external label 'Assignees' and an internal button label stating '2 people'. Lastly, the image shows an action menu with two items: 'Set labels...' and 'Delete...'."
+  src="https://github.com/primer/design/assets/980622/f2867468-4606-4999-8ea7-21860e50051d"
+/>
+
+Select panels can be activated by either a regular [button](/components/button), [icon button](/components/icon-button) or [action menu item](/components/action-menu). If the button is intended to represent the current selection, it's crucial to have a label associated with it, either internally (within the button) or externally (adjacent to the button).
```

Comment:
Clarifying question: Are these intended as examples or a closed set?

Reply:
@siddharthkp good question, I didn't change anything to the existing content here, just condensed it a little. I assume that it was meant to be exclusive to these. Is there anything else that is missing here?